### ISSUE TITLE
add a banner similar to the backup banner for accessing the old wallet

### ISSUE THREAD

**Issue by bbondy** [brave/brave-browser#17995]
It should say something like:
Looking for the deprecated Crypto Wallets extension in Brave? You can still access it here

It should only show if the user has Crypto Wallets setup.

**Comment by UjCbFwtBayFM** [comment]
As the deprecated wallet extension is just a Metamask clone, you could just install MM.

**Comment by bbondy** [comment]
Yes sure you can use MM now and when we have the new wallet. This is for people that specifically have funds already in Crypto Wallets and didn't want to migrate over or are having trouble with the new wallet. 
Btw there's also differences from MetaMask, it's not a blank fork. For example Crypto Wallets can communicate directly with Ledger devices and MetaMask needs to use Ledger Live.

**Comment by bbondy** [comment]
Since this is very prominent we no longer need this (Discussed with James)
<img width="436" alt="Screen Shot 2021-10-07 at 8 02 51 PM" src="https://user-images.githubusercontent.com/831718/136477893-35772369-3146-4b5f-925a-78ea987c510e.png">
